src/ErrorPage/error.jsx

```diff
@@ -2,20 +2,26 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { Redirect } from 'react-router-dom';
+import actions from '../Auth/authActions';
 import ContactHelpdesk from '../lib/ui/contactHelpdesk';
 
 /**
  * Renders the Error page.
  *
  * @returns {Object} JSX representation of the Error page.
  */
-export function ErrorPage({ isAuthenticated, profile }) {
+export function ErrorPage({ isAuthenticated, profile, logout }) {
   const hasAccess = profile.user_metadata && profile.user_metadata.hasAccess;
 
   if (isAuthenticated && hasAccess) {
-    return <Redirect to="/releases" />;
+    return <Redirect to="/dashboard" />;
   }
 
+  const handleLogout = () => {
+    logout();
+    return <Redirect to="/" />;
```

Comment:
this line is not necessary because the logout function already takes you back to the main page